src/lib/utils/data-handlers/resources/resource.ts

```diff
@@ -49,8 +49,9 @@ export function resourceContentsForBookAndChapterFullUrl(
     return apiUrl(resourceContentForBookAndChapter(languageId, bookCode, parseInt(chapter))[0]);
 }
 
-interface ResourceContentInfoWithOccurrences extends ResourceContentInfo {
+export interface ResourceContentInfoWithFrontendData extends ResourceContentInfo {
     occurrences: number;
+    verses: { chapter: number; verse: number }[];
```

Comment:
Might be a result of how the data is, so just is what it is, but it seems like verses would be an array tied to a chapters object, as opposed to repeating the chapter number. I.e. `chapters: { number: number; verses: number[] }[]`.

Reply:
This is meant for helping sort the translation notes. It seemed slightly easier as just a list of chapter/verse instead of a nested list like that.